Modularize, update, and enhance user focus of the "Architecture" part of Planning_for_Project (#2607)
theforeman/foreman-documentation: guides/common/modules/con_organizations-and-locations-in-foreman.adoc

[id="Organizations-and-Locations-in-{ProjectNameID}_{context}"]
= Organizations and locations in {ProjectName}

> On the {ProjectServer}, you can define multiple _organizations_ and _locations_ to help organize content, managed hosts, and configurations.

Reviewer: ```suggestion
On your {ProjectServer}, you can define multiple _organizations_ and _locations_ to help organize content, hosts, and configurations.
```

AFAIK it's never "the ProjectServer"; always "your ProjectServer".

Organizations::

> Organizations typically represent different business units, departments, or teams, such as Finance, Marketing, or Web Development Team.

Reviewer: Do we want/need some emphasis for the examples? I'm leaning towards italics.

Author: IBM Style has a good section on when (not) to add emphasis... but it's not really conclusive for this particular situation. Anyway, let's try it. Applied here and elsewhere.
